bitnami/cert-manager/templates/controller/service.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.metrics.enabled }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: controller
+spec:
+  type: ClusterIP
+  selector: {{- include "common.labels.matchLabels" . | nindent 4 }}
+    app.kubernetes.io/component: controller
+  ports:
+    - port: {{ .Values.controller.containerPort }}
+      protocol: TCP
+      targetPort: {{ .Values.metrics.serviceMonitor.targetPort }}
```

Set a name for the port:
```suggestion
    - name: controller
      port: {{ .Values.controller.containerPort }}
      protocol: TCP
      targetPort: {{ .Values.metrics.serviceMonitor.targetPort }}
```